backend/src/agent/persistence/client/pipeline_client.go

```diff
@@ -23,6 +23,7 @@ import (
 	"google.golang.org/grpc/metadata"
 
 	api "github.com/kubeflow/pipelines/backend/api/v1beta1/go_client"
+	apiv2 "github.com/kubeflow/pipelines/backend/api/v2beta1/go_client"
```

Comment:
Hi @rimolive, you can revert the changes for `apiv2` because we are not planning to change the scheduledworkflow in this PR. This won't break the persistence agent because this piece of code is for sending the workflow update to the scheduled workflow service.

We can open another PR for making scheduledworkflow backend agnostic so it's easier to debug and revert in the future.